deepdrivemd/misc/OutlierDB.py

```diff
@@ -0,0 +1,45 @@
+import os
+import random
+
+
+class OutlierDB:
```

Comment:
Add docstrings and type hinting to document function inputs and class methods.

Reply:
Done